components/toast_wrapper/toast_wrapper.jsx

```diff
@@ -216,20 +216,14 @@ class ToastWrapper extends React.PureComponent {
             return (
                 <FormattedMessage
                     id='postlist.toast.newMessagesSince'
-                    defaultMessage={'{count, number} new {count, plural, one {message} other {messages}} since {date} at {time}'}
+                    defaultMessage='{count, number} new {count, plural, one {message} other {messages}} {isToday, select, true {} other {since}} {date}'
```

Comment:
Have we used `select` anywhere else in translation strings? I'm wondering if we need to add it to some documentation somewhere so that translators know how to translate it. I [started a thread in the I18n channel](https://community-daily.mattermost.com/core/pl/915d8y916tgyjjcn1kwttax1oe) to ask if we have any docs on those.

Reply:
@hmhealey I think this is the first `select` usage. I agree we should add docs... what if we added a Message Syntax section to https://docs.mattermost.com/developer/localization.html with a 1-paragraph overview with links to https://formatjs.io/docs/icu-syntax and https://formatjs.io/docs/react-intl ?

Reply:
Yeah, that would be great. The source for those docs live [here](https://github.com/mattermost/docs/blob/master/source/developer/localization.rst) if you could make a PR

Reply:
[Done](https://github.com/mattermost/docs/pull/3590) ✓